CODEOWNERS

```diff
@@ -0,0 +1,5 @@
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# @global-owner1 and @global-owner2 will be requested for
+# review when someone opens a pull request.
+*       @eddiejaoude @rossanodan
```

Comment:
Is this file needed?

Reply:
With the admin approval thing, I don't think so. I'm going to remove it.

Reply:
ok thanks 👍 - it can go in a new PR, so we can get this merged